test/mv_opentelemetry/ecto_test.exs

```diff
@@ -27,6 +29,32 @@ defmodule MvOpentelemetry.EctoTest do
       :telemetry.detach({[:mv_opentelemetry_harness, :repo], MvOpentelemetry.Ecto, :handle_event})
   end
 
+  test "puts stacktrace when enabled" do
+    MvOpentelemetry.Ecto.register_tracer(
+      tracer_id: :test_ecto_tracer,
+      span_prefix: [:mv_opentelemetry_harness, :repo],
+      default_attributes: [{"service.component", "test.harness"}]
+    )
+
+    config =
+      :mv_opentelemetry_harness
+      |> Application.get_env(MvOpentelemetryHarness.Repo)
+      |> then(&([stacktrace: true] ++ &1))
+
+    Repo.with_dynamic_repo(config, fn ->
+      Page.all() |> Repo.all()
+      assert_receive {:span, span_record}
+      assert {:attributes, _, _, _, attributes} = span(span_record, :attributes)
+      assert %{"ecto.stacktrace" => stacktrace} = attributes
+      assert is_binary(stacktrace)
```

Comment:
Can we make this test a little bit more detailed? `""` is also a binary. 🤔

Reply:
I don't think checking content adds value here because content of the binary is coming from ecto itself so we have no control over it